jina/serve/executors/__init__.py

```diff
@@ -596,6 +599,31 @@ def _add_requests(self, _requests: Optional[Dict]):
                         f'expect {typename(self)}.{func} to be a function, but receiving {typename(_func)}'
                     )
 
+    def _validate_sagemaker(self):
```

Comment:
@JoanFM I moved this logic from `WorkerRequestHandler` to the `Executor` class so the `/invocations` route can access the function bound, if not defined.

Reply:
I do not like this. Executor should not be aware of this. Please move this one to WorkerRequestHandler

Reply:
If `/invocations` route is not defined & only one route is defined in the Executor, I'm adding the function attached to the route to `/invocations` route. Can (Should) WorkerRequestHandler access the function wrapped by a route?

Reply:
okey makes sense, let's leave it here then